src/containers/navigation-icons/user-icons/AccountIcon.jsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+
+import Avatar from '@mui/material/Avatar'
+import NavigationIcon from '~/components/navigation-icon/NavigationIcon'
+import useAxios from '~/hooks/use-axios'
+import { userService } from '~/services/user-service'
+import { useCallback, useEffect } from 'react'
+
+const AccountIcon = ({ buttonProps }) => {
+  const { t } = useTranslation()
+  const { userId, userRole, isUserUpdated } = useSelector(
+    (state) => state.appMain
+  )
+
+  const serviceFunction = useCallback(
+    () => userService.getUserById(userId, userRole),
+    [userId, userRole]
+  )
+
+  const { response, fetchData, loading } = useAxios({
+    service: serviceFunction
+  })
+
+  useEffect(() => {
+    if (isUserUpdated) {
+      fetchData()
+    }
+  }, [isUserUpdated, fetchData])
+
+  const fullName = `${response?.firstName} ${response?.lastName}`
+  const firstLetters = fullName
+    .split(' ')
+    .map((word) => word[0])
+    .join('')
```

Comment:
Are we allowed names like "John Michael Smith"? In this case we show 3 letters and it works fine?

Reply:
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/4d5b7faf-fed8-4c9d-93f1-f54c600f362e)

Yeah, it works with `fullName = John Michael Smith`
Do you think I should take only the first letter of the first and last words?